### DISCUSSION TITLE
 ERR_PNPM_DLX_NO_BIN  No binaries found in cz

### DISCUSSION THREAD

**Discussion by blus-cai** [pnpm/pnpm#7819] [General]
I am installing the commitizen tool，Execute command pnpx cz --pnpm After completion An error occurred.
![20240324014908](https://github.com/pnpm/pnpm/assets/33108751/7580330d-01b8-46b7-9c96-9adae878706e)

**Comment by zkochan** [comment]
Just run `pnpm cz`. `pnpx` is for downloading and executing (see docs: https://pnpm.io/cli/dlx). `pnpm exec` (or just `pnpm exec`) is for executing binaries in `node_modules/.bin` (related docs: https://pnpm.io/cli/exec).
